module16/test_cases.robot

```diff
@@ -0,0 +1,87 @@
+*** Settings ***
+Library    SeleniumLibrary
+Suite Teardown    Close All Browsers
+
+*** Variables ***
+${user_login}    gryundelb@gmail.com
+${user_password}    Password1
+
+*** Test Cases ***
+Test Login
+    [Tags]    screenshot
+    Open Browser    https://www.demoblaze.com/    Chrome
+    Set Selenium Speed    0.5s
+    Maximize Browser Window
```

Comment:
Test environment preparation should not be part of test, it's better use Suite Setup and Test Setup for suite and test env preparation respectively

Reply:
Done. Test Setup is crated.